examples/model_selection/plot_grid_search_stats.py

```diff
@@ -192,7 +192,7 @@ def corrected_std(differences, n_train, n_test):
     """
     n = n_train + n_test
     corrected_var = (
-        np.var(differences, ddof=1) * ((1 / n) + (n_test / n_train))
+        np.var(differences, ddof=1) * ((1 / len(differences)) + (n_test / n_train))
```

Comment:
I think you need to shorten this line (to less than 80 characters) to prevent the linter from failing